### PR TITLE
Add docs for Brottsplatskartan sensor

### PR DESCRIPTION
**Pull request by chrillux**
**Description:**

Brottsplatskartan is a Swedish site for tracking crimes happening all around Sweden. This sensor can be used to track a certain area, e.g where you live.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.brottsplatskartan.markdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+title: "Brottsplatskartan"
+description: "Instructions on how to integrate brottsplatskartan.se into Home Assistant."
+date: 2018-12-02 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Social
+logo: brottsplatskartan.png
+ha_release: 0.84
```
Comment (MartinHjelmare) [inline source/_components/sensor.brottsplatskartan.markdown]: 0.85

File: source/_components/sensor.brottsplatskartan.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### {% linkable_title Area %}
+
+Brottsplatskartan captures all incidents in a region, e.g Stockholms län. If area parameter is defined, any latitude and longitude parameters are ignored.
+
+### {% linkable_title Latitude and Longitude %}
+
+The radius is set to 5 km when using latitude and longitude to monitor an area. It's not possible to explicitly set radius to another value.
+
+### {% linkable_title Events %}
```
Comment (MartinHjelmare) [inline source/_components/sensor.brottsplatskartan.markdown]: This wasn't implemented right?

File: source/_components/sensor.brottsplatskartan.markdown (hunk 1 above)
Comment (chrillux) [reply source/_components/sensor.brottsplatskartan.markdown]: Updated now.

File: source/_components/sensor.brottsplatskartan.markdown (hunk 2 above)
Comment (chrillux) [reply source/_components/sensor.brottsplatskartan.markdown]: No, unfortunately it's not implemented in the API.

Comment (MartinHjelmare) [reply source/_components/sensor.brottsplatskartan.markdown]: Ok, so let's remove it for now.

Comment (chrillux) [reply source/_components/sensor.brottsplatskartan.markdown]: Ahh you meant the "Events", I forgot this was still in the documentation. Removed it now. I thought you were talking about the radius, which is not implemented in the API. :)

**Review by MartinHjelmare — APPROVED** [review]

**Comment by MartinHjelmare** [comment]
Can be merged when build passes.

**Comment by chrillux** [comment]
Cool, thanks!

**Pull request merged**